internal/integration/cli/list.go

```diff
@@ -31,6 +33,79 @@ func (suite *ListSuite) TestSuccess() {
 		base.StdoutShouldNotMatch(regexp.MustCompile(`os-release`)))
 }
 
+// TestDepth tests various combinations of --recurse and --depth flags.
+func (suite *ListSuite) TestDepth() {
+	suite.T().Parallel()
+
+	node := suite.RandomDiscoveredNode(machine.TypeJoin)
+
+	runAndParse := func(flags ...string) []string {
+		args := append([]string{"list", "--nodes", node, "/etc/kubernetes"}, flags...)
```

Comment:
my concern with `/etc/kubernetes` is that it's managed by the kubelet, and its contents might change with `kubelet` version bumps or might change after some interaction with the kubelet

not sure which other directory is better, but my guess `/system/secrets` might be better, as at least it's managed by Talos and its contents are fixed (for control plane nodes only)